lib/src/app/app.dart

```diff
@@ -16,25 +17,151 @@ class AppClient {
 
   AppClient(this._client);
 
+  /// Get the id of the user with the email provided
+  Future<String> getUserIdByEmail(String email) async {
+    final request = GetUserIDByEmailRequest()..email = email;
+    final GetUserIDByEmailResponse response = await _client.getUserIDByEmail(request);
+    return response.userId;
+  }
+
+  /// Create a new [Organization]
+  Future<Organization> createOrganization(String name) async {
+    final request = CreateOrganizationRequest()..name = name;
+    final CreateOrganizationResponse response = await _client.createOrganization(request);
+    return response.organization;
+  }
+
   /// List all the [Organization] the currently authenticated user has access to
   Future<List<Organization>> listOrganizations() async {
     final listOrganizationsRequest = ListOrganizationsRequest();
     final ListOrganizationsResponse response = await _client.listOrganizations(listOrganizationsRequest);
     return response.organizations;
   }
 
+  /// Get all [OrganizationIdentity]s that have access to a [Location].
+  Future<List<OrganizationIdentity>> getOrganizationsWithAccessToLocation(String locationId) async {
+    final request = GetOrganizationsWithAccessToLocationRequest();
```

Comment:
We need to set the `locationId` here.

Reply:
As an aside, it's unfortunate that we don't have a linter to catch unused method params. I wonder if there's an obvious choice for such a thing, or a reason we don't have one set up currently @njooma @clintpurser?

Reply:
There is no linter option for this: https://github.com/dart-lang/linter/issues/2204

Reply:
😭 so it goes!